cerberus/validator.py

```diff
@@ -1189,6 +1189,11 @@ def _validate_forbidden(self, forbidden_values, field, value):
         elif isinstance(value, int):
             if value in forbidden_values:
                 self._error(field, errors.FORBIDDEN_VALUE, value)
+        elif isinstance(value, float):
```

Comment:
please keep it simple. afaict, there's no need to branch off for float or integer values, only for container values that are not strings.

Reply:
I will look at it.